src/main/java/seedu/address/model/person/ClassId.java

```diff
@@ -33,7 +33,7 @@ public ClassId(String classId) {
      * Returns true if a given string is a valid grade.
      */
     public static boolean isValidClassId(String test) {
-        return test.matches(VALIDATION_REGEX);
+        return test.matches(VALIDATION_REGEX) && test.contains("B");
```

Comment:
1. Would `startsWith()` be a better option instead of `contains()`? Since a class id such as "37B" would not be considered valid.

2. Should we also check if the class id has a length of exactly 3? Since a class id such as "B1" would not be valid. Or maybe we could convert single digits into double digits by prepending a 0 in front?

Reply:
good suggestion! i think we can enforce exactly length of 3 and I can update the error message to tell the user that we only accept B01 or B11 etc